lib/zclient.c

```diff
@@ -934,9 +934,27 @@ enum zclient_send_status zclient_send_rnh(struct zclient *zclient, int command,
 enum zclient_send_status
 zclient_route_send(uint8_t cmd, struct zclient *zclient, struct zapi_route *api)
 {
+	enum zclient_send_status ret;
```

Comment:
Looking at this code now,  I realize I pointed you slightly in the wrong direction.  Let's change this so that if the old_buf size is too small to allocate a new one for the zclient->obuf and just swap them?